documentation/polymer-templates/tutorial-template-components-in-slot.asciidoc

```diff
@@ -48,28 +49,33 @@ public class ComponentContainer extends PolymerTemplate<TemplateModel> {
 }
 
 ----
-
-Without the *slot* tag in the template the added label and button would not be visible to the user even though it can be located in the DOM.
-As we can see multiple components will show up in the slot when added to the template element that has an open `<slot></slot>`
+* Without the `<slot>` tag in the template, the label and button would not be visible to the user, even though they can be located in the DOM.
+* You can add multiple components and they will display in the slot when added to the template element that has an open `<slot></slot>`.
 
 [NOTE]
-Any element added to the light dom to be shown inside a `<slot>` can be removed from the dom and it will work as expected and disappear from the main element.
+You can remove any element from a `<slot>` and it will work as expected, and no longer be displayed in the main element.
+
+== Using Default Content in the <slot> Element
 
-== Default content slots and named slots in templates
+The `<slot>` tag can contain default content that displays if nothing is set from the light DOM.
 
-The `<slot>` tag can contain "default" content if nothing is set from the light dom.
+*Example*: `<slot>` element with default content.
 
-For example the template could be built as:
 [source,html]
 ----
 <div style="border: 1px solid black; padding: 10px; margin: 10px;">
     <slot>No components added</slot>
 </div>
 ----
 
-This would show at start 'No components added' in the slot which would then be replaced by the first added child element.
+* 'No components added' displays initially, and is then replaced by the first added child element.
```

Comment:
> replaced by the first added child element.

Only the first child element, or all child elements? Or perhaps do you mean that any added child element replaces the 'No components added'?

Reply:
I believe the original author meant that the "no components added" text will be replaced as soon as there is at least one child element added, hence the "first". Though it is worded in a way that may cause others to get confused. Will modify it, thanks for pointing it out.